bridge/opentracing/internal/mock.go

```diff
@@ -5,7 +5,8 @@ package internal // import "go.opentelemetry.io/otel/bridge/opentracing/internal
 
 import (
 	"context"
-	"math/rand"
+	crand "crypto/rand"
```

Comment:
Similar here regarding the import of `crypto/rand`.

Reply:
I went with a simpler seeding strategy for this one.
It does need ChaCha8 rather than PCG because it uses Read() which only ChaCha8 does.